src/org/parosproxy/paros/extension/report/ExtensionReport.java

```diff
@@ -82,53 +85,43 @@ public void hook(ExtensionHook extensionHook) {
 	private ZapMenuItem getMenuItemHtmlReport() {
 		if (menuItemHtmlReport == null) {
 			menuItemHtmlReport = new ZapMenuItem("menu.report.html.generate");
-			menuItemHtmlReport.addActionListener(new java.awt.event.ActionListener() { 
-
-				@Override
-				public void actionPerformed(java.awt.event.ActionEvent e) {    
-
-				    ReportLastScan report = new ReportLastScan();
-				    report.generateReport(getView(), getModel(), ReportLastScan.ReportType.HTML);
-	                
-				}
-			});
-
+            menuItemHtmlReport.addActionListener((java.awt.event.ActionEvent e) -> {

```

Comment:
- The type is not needed, it could also be changed to one statement, e.g.:
```Java
menuItemHtmlReport.addActionListener(e -> new ReportLastScan().generateReport(...));
```

 - It would be better if the code kept the same indentation as surrounding code.